lms/static/scripts/frontend_apps/components/ValidationMessage.js

```diff
@@ -34,9 +34,12 @@ export default function ValidationMessage({
   });
 
   return (
-    <button onClick={closeValidationError} className={errorClass}>
-      {message}
-    </button>
+    <input
+      type="button"
+      onClick={closeValidationError}
+      className={errorClass}
+      value={message}
+    />
```

Comment:
This works, although you can set the `type` attribute on the `button` as well rather than change to `<input type="button">`. [According to MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/button) `<input type="button">` still works, although using a `<button>` element is preferred.

Reply:
Initially i tried that and I don't think it worked, but I will double check. Its quite possible I did something wrong and then just moved on to <input type=""